warmup.py

```diff
@@ -117,7 +135,24 @@ def force(self, r):
             If ``r`` is 0, the force is :py:obj:`numpy.inf`.
 
         """
-        pass
+        dr = 1e-8
```

Comment:
In general, this will work too! However, we can actually compute the force analytically, which is more accurate than finite differencing. Can you work out the formula for `F` based on `U`? Then, apply the same comments as above to compute these forces analytically.